### PR TITLE
robert king quiz

### PR DESCRIPTION
**Pull request by rob-king**

### CODE REVIEW

**Comment by rob-king** [comment]
I accidently left out an equal sign in the lion class.

File: README.md [hunk 1]
```diff
@@ -147,22 +175,25 @@ CREATE TABLE persons(
...
 
 Your answer:
-```
-Replace this with your answer
+```ruby
+get '/oh_hello' do
+  "Hello world"
+end
+
 ```
```
Comment (superbuggy) [inline README.md:199]: 👍

File: README.md [hunk 2]
```diff
@@ -147,22 +175,25 @@ CREATE TABLE persons(
...
 
 Write ruby code that will query for any person that is 15 years of age
 
 Your answer:
-```
-Replace this with your answer
+```ruby
+Person.where(age:15)
 ```
```
Comment (superbuggy) [inline README.md:188]: 👍

File: README.md [hunk 3]
```diff
@@ -125,7 +153,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema describes how a database is constructed. In a database, a entries on one table would match multiple tables on another database through the primary key (foreign key on the other table).
 ```
```
Comment (superbuggy) [inline README.md:158]: the foreign keys would match the ids of _entries_ (a row in a table) in other tables. your def of schema is 💯 correct tho; more specifically it defines what columns in a table

File: README.md [hunk 4]
```diff
@@ -114,7 +138,11 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+Person might have the following attributes: name, age, and birth_place.
+A person could have a one to many relationship with Pet and Lamp.
+A Genie could have the following attributes: name, age, wishes_granted.
+A Genie could have a one to one relationship with a Lamp
+A pet could have the following attributes: name, age, spcies
 ```
```
Comment (superbuggy) [inline README.md:147]: 👍

File: README.md [hunk 5]
```diff
@@ -97,7 +100,28 @@ Each lion should have:
...
+  def greet
+    puts "Hello, I am #{@name}"
+  end
+end
+
+class Lion < Animal
+	attr_reader :king
+	def initialize(name)
+		@name = name.capitalize
+		@king = (@name = "Simba") ? true:false
```
Comment (superbuggy) [inline README.md:119]: sick ternary

File: README.md [hunk 6]
```diff
@@ -72,7 +75,7 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends.each { |friend| puts "Belle is friends with #{friend}"  }
 ```
 ## Ruby OOP (meets Lion King)
```
Comment (superbuggy) [inline README.md:80]: 👍

File: README.md [hunk 7]
```diff
@@ -50,7 +52,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+castle[:guests] << residents[1]
+residents.delete("Belle")
 ```
```
Comment (superbuggy) [inline README.md:58]: you'd have to add `residents = town[:residents]` and `castle = town[:castle]`

File: README.md [hunk 8]
```diff
@@ -27,7 +27,9 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def offer_rose(person)
+  puts "Would you take this rose, #{person}?"
+end
 ```
```
Comment (superbuggy) [inline README.md:34]: 👍

**Comment by superbuggy** [comment]
👍 overall

**Pull request closed without merging**